src/common.jl

```diff
@@ -1,11 +1,11 @@
-struct OperatorAssumptions{issquare} end
+struct OperatorAssumptions{issq} end
 function OperatorAssumptions(issquare = nothing)
-    issquare = something(_unwrap_val(issquare), Nothing)
-    OperatorAssumptions{issquare}()
+    issq = something(_unwrap_val(issquare), Nothing)
+    OperatorAssumptions{issq}()
 end
-issquare(::OperatorAssumptions{issq}) where {issq} = issq
+SciMLOperators.issquare(::OperatorAssumptions{issq}) where {issq} = issq
```

Comment:
why is this the same function as the operators one? They don't seem to really do the same thing.

Reply:
`SciMLOperators.issquare(A)` says whether a matrix is square or not. `LinearSolve.issquare(assumptions)` says whether the assumption is for a square matrix or not. pretty much the same thing. If you don't think so LMK and I'll change `LinearSolve.issquare` to `LinearSolve._issquare` to avoid name conflict

Reply:
OperatorAssumptions is not an operator, so it's a little weird. I'd keep them separate and do `__issquare`

Reply:
sure

Reply:
done